src-gui/src/renderer/components/alert/DaemonStatusAlert.tsx

```diff
@@ -107,3 +131,36 @@ export default function DaemonStatusAlert() {
       return exhaustiveGuard(contextStatus);
   }
 }
+
+export function BackgroundProgressAlerts() {
+  const backgroundProgress = usePendingBackgroundProcesses();
+  const classes = useStyles();
+
+  if (backgroundProgress.length === 0) {
+    return null;
+  }
+
+  const componentCounts: Record<string, number> = {};
+  backgroundProgress.forEach(([, status]) => {
+    componentCounts[status.componentName] = (componentCounts[status.componentName] || 0) + 1;
+  });
+
+  const renderedComponentNames = new Set<string>();
+  const uniqueBackgroundProcesses = backgroundProgress.filter(([, status]) => {
+    if (!renderedComponentNames.has(status.componentName)) {
+      renderedComponentNames.add(status.componentName);
+      return true;
+    }
+    return false;
+  });
+
+  return uniqueBackgroundProcesses.map(([id, status]) => (
+    <PartialInitStatus
+      key={id}
+      status={status}
+      classes={classes}
+      currentNumOfType={1} // Since we only show the first one
+      totalOfType={componentCounts[status.componentName]}
+    />
+  ));
```

Comment:
Maybe this is a bit over complicated? The purpose of this is to not display multiple Alerts if we have multiple simultaneous Bitcoin sync processes running